source/include/core_mqtt_agent.h

```diff
@@ -118,7 +118,7 @@ typedef struct MQTTAgentCommandContext MQTTAgentCommandContext_t;
  * application wants to enqueue command(s) with non-zero blocking time, the
  * callback can notify a different task to enqueue command(s) to the MQTT agent.
  */
-typedef void (* MQTTAgentCommandCallback_t )( void * pCmdCallbackContext,
+typedef void (* MQTTAgentCommandCallback_t )( MQTTAgentCommandContext_t * pCmdCallbackContext,
```

Comment:
Can you also update the doxygen examples for the type?

Reply:
Are they not updated here?

Reply:
Oops, I didn't notice that most of the changes were related to doxygen updates. 😅